.github/workflows/ci.yml

```diff
@@ -25,6 +25,43 @@ jobs:
 
       - name: Run verification
         run: make check
+  e2e:
+    name: E2E
+    needs: build_all
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Download all artifacts
+        uses: actions/download-artifact@v3
+      -
+        name: Load Docker images
+        run: |
+          docker load --input kubeclarity/kubeclarity.tar
+          docker load --input runtime_k8s_scanner/runtime_k8s_scanner.tar
+          docker load --input kubeclarity-sbom-db/kubeclarity-sbom-db.tar
+          docker load --input cis_docker_benchmark_scanner/cis_docker_benchmark_scanner.tar
+          docker image ls -a
+      - name: Edit values.yaml
+        id: edit_values
+        run: |
+          tag=${{ github.sha }}
+          sed -i 's/latest/'"$tag"'/g' charts/kubeclarity/values.yaml
```

Comment:
please use the same as in `release.yaml`
```
      find . -type f -name "values.yaml" -o -name "Chart.yaml" |
          xargs sed -i -e s/latest/${{ github.sha }}/g
```

Reply:
Not working for some reason: https://github.com/openclarity/kubeclarity/runs/6558658017?check_suite_focus=true

Reply:
```
find . -type f -name "values.yaml" -o -name "Chart.yaml" | xargs sed -i -e s/latest/41b9c21a81e7fc8618cba764dc7063a9c9329485/g sed -i 's/Always/IfNotPresent/g' charts/kubeclarity/values.yaml
```
`sed -i 's/Always/IfNotPresent/g' charts/kubeclarity/values.yaml` should run in a different line